x-pack/test/functional/services/ml/job_wizard_common.ts

```diff
@@ -628,5 +628,69 @@ export function MachineLearningJobWizardCommonProvider(
         await testSubjects.existOrFail(expectedSelector);
       });
     },
+
+    async assertAnnotationRecommendationCalloutVisible(expectVisible: boolean = true) {
+      const callOutTestSubj = 'mlJobWizardAlsoEnableAnnotationsRecommendationCallout';
+      if (expectVisible)
+        await testSubjects.existOrFail(callOutTestSubj, {
+          timeout: 3_000,
+        });
+      else
+        await testSubjects.missingOrFail(callOutTestSubj, {
+          timeout: 3_000,
+        });
+    },
+
+    async goToTimeRangeStep() {
+      await retry.tryForTime(60_000, async () => {
+        await testSubjects.existOrFail('mlJobWizardTimeRangeStep');
+        await testSubjects.click('mlJobWizardTimeRangeStep');
+        await testSubjects.existOrFail('mlJobWizardStepTitleTimeRange');
+      });
+    },
+
+    async setShortDurationTimeRange(shortDurationEndDate: string) {
```

Comment:
```suggestion
    async setTimeRange(shortDurationEndDate: string) {
```
it's not necessarily a short duration, isn't it? shall we provide a generic name?